api/migrations/20240715122249-fix_classes_ile-de-france.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  async up(db) {
+    db.collection("etablissements").updateMany({ region: "Ile-de-France" }, { $set: { region: "Île-de-France" } });
+
+    db.collection("classes").updateMany({ region: "Ile-de-France" }, { $set: { region: "Île-de-France" } });
```

Comment:
il manque juste le await : 

```suggestion
    await db.collection("etablissements").updateMany({ region: "Ile-de-France" }, { $set: { region: "Île-de-France" } });

    await db.collection("classes").updateMany({ region: "Ile-de-France" }, { $set: { region: "Île-de-France" } });
```